src/header/styles.scss

```diff
@@ -137,6 +137,7 @@
     // whether there are buttons present or not; otherwise configurable dashboard handles are misaligned.
     &.refresh {
       padding-top: space-heading-button-diff(awsui.$line-height-heading-xl);
+      color: awsui.$color-text-heading-primary;
```

Comment:
Should we apply this to Classic as well? Classic inherits the design tokens from Visual Refresh, which means that Classic users will also see this `color-text-heading-primary` token listed, with the description "The default color for heading 1 (...)", so they will expect to be able to theme primary headings with it but it will have no effect for them.